lib/tokenizers.js

```diff
@@ -256,6 +280,20 @@ async function _createTokenizer() {
       type: key.type
     }
   };
+  const collection = database.collections['tokenizer-tokenizer'];
+
+  if(explain) {
+    const resultMap = new Map();
+    const cursor = await collection.find(query);
+    resultMap.set('queryCursor', await cursor.explain('executionStats'));
```

Comment:
Can we more specific here and elsewhere? When using a map here, we have an opportunity to be more descriptive. If one were to wrap these database calls into their own functions, what would they be named? That mental model may provide a good starting point for what to name these properties.

In this case, we can rename `queryCursor` to `getPendingTokenizerById`.